commands

```diff
@@ -35,9 +35,6 @@ case "$1" in
     ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_ROOT/data:/data" -v "$SERVICE_ROOT/config:/usr/local/etc/redis" -d --restart always --label dokku=service --label dokku.service=redis "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
     echo "$ID" > "$SERVICE_ROOT/ID"
 
-    dokku_log_verbose_quiet "Waiting for container to be ready"
-    sleep 10
-
```

Comment:
It would be better if we could really wait for the container to be ready. But I’m not sure how to do this (or if it is possible).
I’m not a fan of having code only for tests but in this case our test suite will take forever with all these `sleep` calls.